docs/aerodromes/classd/Archerfield.md

```diff
@@ -86,6 +144,12 @@ The APCH field should include `EXP INST APCH` when:
   - the ceiling is at or below `A029`; or  
   - visibility is less than 5000m  
 
+### Helicopter Area
+The ATIS must indicate the current active helicopter area, refer to [Helicopter Areas](#areas).
+
+!!! example  
+    OPR INFO: RWY 04, 22, AND TWY C AND D UAVBL. **HELICOPTER AREA A ACTIVE**  
```

Comment:
Are runway 04/22 and taxiways C/D dependent on heli area A being active, or is this just an example?

Reply:
04/22 and C/D don't have any correlation to the active helicopter areas, this was just an example as about 90% of the time you'll hear this in the real world ATIS as the grass runways and their taxiways aren't being used, followed by the statement of which helicopter area was active.

Reply:
Ah ok, easy. We try to make all of the examples super simple and relevant so that controllers can see the direct effect of the procedure described. Perhaps we could remove the unavailable runway/taxiway comments so that the OPR INFO is just the heli area?

Reply:
Yeah will do

Reply:
Fixed